### PR TITLE
replaced appcelerator urls with tidev

### PR DESCRIPTION
**Pull request by caspahouzer**

### CODE REVIEW

File: docs/guide/Titanium_SDK/Titanium_SDK_How-tos/User_Interface_Deep_Dives/iOS_UI_Components_and_Conventions/README.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ With 1.8, the location of the Settings.bundle was changed. Platform-specific res
 
 If you're running SDK 1.8+
 
-1. Download a recent copy of the [KitchenSink app](https://github.com/appcelerator/KitchenSink)
+1. Download a recent copy of the [KitchenSink app](https://github.com/tidev/KitchenSink)
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_How-tos/User_Interface_Deep_Dives/iOS_UI_Components_and_Conventions/README.md]: Bad link. The correct link is https://github.com/tidev/kitchensink-v2. There are multiple instances, so grep to find them all. Thanks!

**Review by cb1kenobi — CHANGES_REQUESTED** [review]

File: docs/guide/Titanium_SDK/Titanium_SDK_How-tos/User_Interface_Deep_Dives/iOS_UI_Components_and_Conventions/README.md (hunk 1 above)
Comment (caspahouzer) [reply docs/guide/Titanium_SDK/Titanium_SDK_How-tos/User_Interface_Deep_Dives/iOS_UI_Components_and_Conventions/README.md]: 7 replacements in 4 files

File: docs/guide/Titanium_SDK/Titanium_SDK_Guide/Example_Applications.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ Kitchen Sink 2.0 is a refactored Alloy-based version of the KitchenSink demo-app
 
 ![kitchensink-overview](./kitchensink-overview.png)
 
-[Kitchen Sink 2.0 on GitHub](https://github.com/tidev/kitchensink-v2)
+[Kitchen Sink 2.0 on GitHub](https://github.com/tidev/kitchensink-v2-v2)
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_Guide/Example_Applications.md]: v2-v2?

File: docs/guide/Alloy_Framework/Alloy_How-tos/Alloy_Samples.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ weight: '40'
 
 ## Kitchen Sink
 
-Want to check out Alloy Samples in an actual app? Look at the [KitchenSink-v2 application on GitHub](https://github.com/tidev/kitchensink-v2).
+Want to check out Alloy Samples in an actual app? Look at the [KitchenSink-v2 application on GitHub](https://github.com/tidev/kitchensink-v2-v2).
```
Comment (cb1kenobi) [inline docs/guide/Alloy_Framework/Alloy_How-tos/Alloy_Samples.md]: v2-v2?

**Review by cb1kenobi — CHANGES_REQUESTED** [review]

File: docs/guide/Alloy_Framework/Alloy_How-tos/Alloy_Samples.md (hunk 3 above)
Comment (caspahouzer) [reply docs/guide/Alloy_Framework/Alloy_How-tos/Alloy_Samples.md]: copy, paste, destroyed.. sorry

**Review by cb1kenobi — APPROVED** [review]
Looks good! **APPROVED**!

**Pull request merged**